src/css.js

```diff
@@ -130,11 +130,12 @@ function boxModelAdjustment( elem, dimension, box, isBorderBox, styles, computed
 
 	// Account for positive content-box scroll gutter when requested by providing computedVal
 	if ( !isBorderBox && computedVal >= 0 ) {
+		var val = elem[ "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 ) ];
```

Comment:
The `offsetWidth`/`offsetHeight` comment seems misplaced now, doesn't it?

Also, a comment about why the fallback is needed (below, at line 138) would be useful, it's not clear now.